compiler/luci/pass/src/SubstitutePadV2ToPadPass.cpp

```diff
@@ -93,6 +93,8 @@ struct Paddings
   /**
    * @brief Store paddings position information.
    * @details _padding_pos[k] stores Pad object at axis k
+   *
+   * @note  Paddings must be for rank 4 tensor
```

Comment:
Now, class `Paddings` support tensor with rank-4 only and `assert` exists only in the constructor.